pylintrc

```diff
@@ -7,19 +7,28 @@ extension-pkg-whitelist=
 
 # Add files or directories to the denylist. They should be base names, not
 # paths.
-ignore=build,protos,sdk,db_migrations,temporary_db_migrations_for_pre_1_users
-
+ignore=
```

Comment:
Switched to using `ignore-paths` which is more accurate because it matches against paths, not base names.